e2e/new/tests/build/build.go

```diff
@@ -1,10 +1,72 @@
 package build
 
-import "github.com/onsi/ginkgo"
+import (
+	"context"
+	"errors"
+	"os"
+
+	"github.com/docker/docker/api/types"
+	"github.com/loft-sh/devspace/cmd"
+	"github.com/loft-sh/devspace/cmd/flags"
+	"github.com/loft-sh/devspace/e2e/new/framework"
+	"github.com/loft-sh/devspace/pkg/devspace/docker"
+	"github.com/loft-sh/devspace/pkg/util/factory"
+	"github.com/loft-sh/devspace/pkg/util/log"
+	"github.com/onsi/ginkgo"
+)
+
+var _ = DevSpaceDescribe("build", func() {
+
+	initialDir, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+
+	// create a new factory
+	var f factory.Factory
+
+	// create logger
+	var log log.Logger
+
+	// create context
+	ctx := context.Background()
+
+	ginkgo.BeforeEach(func() {
+		f = framework.NewDefaultFactory()
+	})
 
-var _ = DevSpaceDescribe("dependencies", func() {
 	ginkgo.It("should build dockerfile with docker", func() {
-		// TODO
+		tempDir, err := framework.CopyToTempDir("tests/build/testdata")
```

Comment:
Would be good to create the testdata in `tests/build/testdata/docker` so that we can add other subfolders for other tests as well

Reply:
done.